pages/reference/deployed-contract-addresses.en-US.mdx

```diff
@@ -6,26 +6,35 @@ import { Callout } from 'nextra-theme-docs';
 
 # Contract Addresses
 
-Addresses of deployed contracts of the Livepeer protocol on various networks. All contracts implement the Streamflow version of the Livepeer protocol.
+Addresses of deployed contracts of the Livepeer protocol on various networks. All contracts implement the Delta version
```

Comment:
Suggest modifying to:

"The Arbitrum Mainnet and following Mainnet contracts implement the Delta version of the protocol"

"...list Mainnet contracts"

"Since the Confluence upgrade, all Mainnet contracts besides the above are paused"